### PR TITLE
Add support for createCustomerProfileFromTransaction

### PR DESCRIPTION
**Pull request by mpuittinen**
Hi!

Added support for createCustomerProfileFromTransaction and fixed some issues with test cases not succeeding. Please note package.json refers to auth-net-request from our github account since it requires the changes done there as well. That will need to be fixed once the updated auth-net-request module is published.

Cheers,

Mikael

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,11 +24,12 @@
   "author": "Daniel Durante <me@danieldurante.com>",
   "license": "MIT",
   "dependencies": {
-    "auth-net-request": "2.2.2",
+    "auth-net-request": "SC5/authorize-net-request",
```
Comment (durango) [inline package.json]: Can you revert this line please?

Comment (mpuittinen) [inline package.json]: Will the new package of auth-net-request be 2.2.3? The change won't work with the 2.2.2 package.

-Mikael

Comment (durango) [inline package.json]: yup, I believe I just updated it, but we'll still need the request dependency below

Comment (mpuittinen) [inline package.json]: Hi!

I’ve updated the version to 2.2.3 in our fork and verified that it builds (with a clean node_modules) & passes tests correctly.

-Mikael

**Comment by mpuittinen** [comment]
Hi!

I just noticed that I had forgotten to update the README to include the new API. I'll add that tomorrow.

-Mikael

**Comment by durango** [comment]
Would definitely appreciate it thanks :)

**Comment by mpuittinen** [comment]
Done :)

**Comment by durango** [comment]
Could you open a new PR whenever you get the chance?

**Pull request merged**